### ISSUE TITLE
can't use in Angular2 2.0.2

### ISSUE THREAD

**Issue by goleary** [brtnshrdr/angular2-image-zoom#4]
does this only work with RC builds?

getting the following error:
       ```
           >> Error: Unexpected value 'ImageZoomModule' imported by the module 'PopupAppModule'

> > ```
> > at C:\Users\golear\Documents\GitHub\ShoppingAssist\node_modules\@angular\compiler\bundles\compiler.umd.js:14126:37
> > at Array.forEach (native)
> > at CompileMetadataResolver.getNgModuleMetadata (C:\Users\golear\Documents\GitHub\ShoppingAssist\node_modules\@angular\compiler\bundles\compiler.umd.js:14111:46)
> > at C:\Users\golear\Documents\GitHub\ShoppingAssist\node_modules\@angular\compiler\bundles\compiler.umd.js:12952:58
> > at Array.forEach (native)
> > at OfflineCompiler.analyzeModules (C:\Users\golear\Documents\GitHub\ShoppingAssist\node_modules\@angular\compiler\bundles\compiler.umd.js:12951:21)
> > at CodeGenerator.codegen (C:\Users\golear\Documents\GitHub\ShoppingAssist\node_modules\@angular\compiler-cli\src\codegen.js:105:47)
> > at codegen (C:\Users\golear\Documents\GitHub\ShoppingAssist\node_modules\@angular\compiler-cli\src\main.js:7:81)
> > at Object.main (C:\Users\golear\Documents\GitHub\ShoppingAssist\node_modules\@angular\tsc-wrapped\src\main.js:30:16)
> > at Object.<anonymous> (C:\Users\golear\Documents\GitHub\ShoppingAssist\node_modules\@angular\compiler-cli\src\main.js:14:9)
> > ```
> > 
> > Compilation failed
> > Exited with code: 1.
> > Warning: Task "exec:ngc" failed. Use --force to continue.

Aborted due to warnings.

```
```

**Comment by tedkhg** [comment]
I had a same problem at Angular2 2.0.1
I used it with Ionic2, but it failed at build time.

**Comment by brtnshrdr** [comment]
Issue has been fixed in version 1.1.1
